app/assets/javascripts/mumuki_laboratory/application/uppercase-mode.js

```diff
@@ -0,0 +1,5 @@
+mumuki.load(() => {
+  if (JSON.parse($('#mu-uppercase-mode-enabled').val())) {
```

Comment:
Why having this `if` and a `#mu-uppercase-mode-enabled` instead of just enabling `mu-uppercase` directly in `application.html.erb`?

Reply:
True. Thought I'd had to do more stuff on JS, turns out it was only one if. I'll do that.